Automattic/woocommerce-payments
# Visual fixes for the connect page hero

Fixes #6780 

#### Changes proposed in this Pull Request

* Replace the connect hero illustration with the right one
* Style the connect hero illustration more in line with the new designs (alignment)
* Fix a couple of wrong font sizes for the connect hero
* Add `border-radius` to payment icons
* Optimize a .png image

Before:
![Screenshot 2023-07-20 at 15 26 52](https://github.com/Automattic/woocommerce-payments/assets/8830539/469f03f8-93f3-4732-b746-2262cfc2e18b)

After:
![Screenshot 2023-07-20 at 15 24 10](https://github.com/Automattic/woocommerce-payments/assets/8830539/87c3d92d-509f-4c56-b8e2-04eb353fec91)

#### Testing instructions

* Checkout the PR branch on your local client installation
* Build the client with `npm run build:client`
* Force WCPay to act as disconnected via the WCPay Dev Tools
* Check out the Payments connect page. It should look like in the screenshot above.

-------------------


**Post merge**
